### PR TITLE
Prefix tuning experiment

### PR DESCRIPTION
**Pull request by dirkgr**
Like the readme says, this is a proof of concept more than a fully-fledged experiment.

### CODE REVIEW

**Comment by dirkgr** [comment]
@AkshitaB, this is ready for review, at least after https://github.com/allenai/tango/pull/231 is merged.

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ai2-tango[torch,transformers,fairscale,beaker,wandb]>=1.0
-#ai2-tango[torch,transformers,fairscale,beaker,wandb] @ git+https://github.com/allenai/tango.git@main
+#ai2-tango[torch,transformers,fairscale,beaker,wandb]>=1.0
+ai2-tango[torch,transformers,fairscale,beaker,wandb] @ git+https://github.com/allenai/tango.git@89980c21966e68bdab28a21f18b84b81a0dffb60
```
Comment (AkshitaB) [inline requirements.txt:2]: Why do we need a specific commit?

Comment (AkshitaB) [reply requirements.txt:2]: Never mind, I'm reviewing the tango PR now.

**Review by AkshitaB — APPROVED** [review]

**Comment by dirkgr** [comment]
🎉

**Pull request merged**